src/util/omit.test.js

```diff
@@ -0,0 +1,19 @@
+import omit from './omit'
+
+describe('omit', () => {
```

Comment:
- Add a test which checks that when trying to omit all values in the object, it returns an empty object.
- Add a test which checks that nothing goes wrong when trying to omit a value which isn't present in the object.